### ISSUE TITLE
all screens: use Fullscreen API when available

### ISSUE THREAD

**Issue by herebefrogs** [herebefrogs/forex-hockey#26]
to maximize playable area without having to save the page as a desktop shortcut

**Comment by Wawet76** [comment]
On Firefox, the url bar reappears when shooting pucks down. Switching to fullscreen will probably avoid that.

**Comment by herebefrogs** [comment]
Yes that's annoying like hell. And the game doesn't stop on Firefox when you tilt the phone in landscape mode, but that's more related to [Issue #27](https://github.com/herebefrogs/forex-hockey/issues/27)
